src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -745,7 +745,7 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
   /** Returns True if person name or org name has changed from its original properties. */
   protected hasNameChanged (orgPerson: OrgPersonIF): boolean {
     // is this a pre-existing person?
-    if (this.isPreExisting && this.isPerson) {
+    if (this.isFirmCorrectionFiling && this.isPreExisting && this.isPerson) {
```

Comment:
Before - change first name for CCC. 
![image](https://user-images.githubusercontent.com/116035339/233726335-d19dbc15-39c3-4263-9fd8-28dd4d134367.png)

After - change first name for CCC.
![image](https://user-images.githubusercontent.com/116035339/233726592-6c3e8582-bc7b-4a27-a263-65bcd13f820e.png)

Checkbox still displayed for firms.
![image](https://user-images.githubusercontent.com/116035339/233726768-cc64ab36-432a-405b-8cc7-9a015fe21cbf.png)